eng/Versions.props

```diff
@@ -121,5 +124,6 @@
   </PropertyGroup>
   <PropertyGroup>
     <MicrosoftDotNetWpfDncEngVersion>10.0.0-alpha.1.24474.3</MicrosoftDotNetWpfDncEngVersion>
+    <MicrosoftWindowsCsWin32PackageVersion>0.3.106</MicrosoftWindowsCsWin32PackageVersion>
```

Comment:
Fyi: Not actually using this in this PR. Now that I have the tests figured out, I'll be creating another updated PR converting the SplashScreen to using CsWin32 and the shared assembly coming from WinForms. Noticed I had left it in after I created the PR.